tests/integration-service/README.md

```diff
@@ -1,92 +1,122 @@
+
+### 4. Happy Path Tests within `integration-with-env.go`
+Checkpoints:
+- Creating an IntegrationTestScenario pointing to an integration pipeline with environment settings.
+- Successfully creating applications and components.
+- Verifying that BuildPipelineRuns are triggered, contain finalizers, and finish successfully.
+- Asserting the signing of the BuildPipelineRun and successful Snapshot creation.
+- Ensuring that the Integration PipelineRuns finish successfully and report status to the Snapshot.
+- Verifying that snapshots are marked as 'passed' after Integration PipelineRuns finish.
+- Ensuring that CronJobs such as "spacerequest-cleaner" exist.
+- Handling space requests in the namespace and ensuring they are created correctly.
+- Verifying proper cleanup after successful deletion of the Integration PipelineRun.
+
+---
+
+## Negative Tests
+
+Negative testing focuses on how the system behaves under invalid, unexpected, or failing conditions to ensure robustness and error handling.
+
+### 1. Negative Test Cases within `integration.go`
+Checkpoints:
+- Creating an IntegrationTestScenario that is expected to fail.
+- Asserting that a snapshot is marked as failed.
 - Asserting that a snapshot is still marked as failed.
-- Validating that no Release CRs is created in certain scenarios.
+- Validating that no Release CRs are created in certain scenarios.
 - Checking that the global candidate does not get updated unexpectedly.
 
-  
-## Running e2e tests
+### 2. Negative Test Cases within `status-reporting-to-pullrequest.go`
+Checkpoints:
+- Creating two IntegrationTestScenarios: one that should fail.
+- Verifying that failed Integration PipelineRuns are reported correctly in the PR's CheckRun.
+- Checking that snapshots are marked as 'failed' if any test fails.
 
-- [Prepare the cluster.](https://github.com/redhat-appstudio/e2e-tests#install-appstudio-in-e2e-mode)
+### 3. Negative Test Cases within `gitlab-integration-reporting.go`
+Checkpoints:
+- Creating two IntegrationTestScenarios: one expected to fail.
+- Verifying that the failing Integration PipelineRun is reflected as "Fail" in the MR's CommitStatus.
+- Ensuring that MR notes show the failure status of the integration test.
+- Asserting that no releases are triggered if any integration test fails.
+
+### 4. Negative Test Cases within `integration-with-env.go`
+Checkpoints:
+- Verifying that integration pipelines are marked as failed when tests do not pass.
+- Checking that snapshots are marked as 'failed' when Integration PipelineRuns do not finish successfully.
+- Ensuring that space requests are deleted correctly in failed scenarios.
+
+---
 
+## Running E2E Tests
 
-- To run service-level e2e test suite: integration.go
+- [Prepare the cluster.](https://github.com/redhat-appstudio/e2e-tests#install-appstudio-in-e2e-mode)
+
+- To run service-level E2E test suite: `integration.go`
 
 ```bash
 $ ./bin/e2e-appstudio --ginkgo.focus="integration-service-suite" –ginkgo.vv
-```
```

Comment:
Should we re-add this since it's formatting this command weirdly?

Reply:
Will do. I want to double check the content anyway. I'll get the last push in later today, once I've checked it all over and fixed up that formatting thing.